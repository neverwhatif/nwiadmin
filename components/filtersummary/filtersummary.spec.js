import React from 'react';
import { shallow } from 'enzyme';

import FilterSummary from './index';

const mockLocation = {
    pathname: '/lorem',
    search: '',
};

describe('FilterSummary', () => {
    const component = shallow(<FilterSummary location={mockLocation} />);

    it('should render', () => {
        expect(component).toMatchSnapshot();
    });
});
