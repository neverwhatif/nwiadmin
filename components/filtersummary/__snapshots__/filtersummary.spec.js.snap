// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterSummary should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FilterSummary
    filterMap={Object {}}
    location={
      Object {
        "pathname": "/lorem",
        "search": "",
      }
    }
    resetFilters={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): null,
  Symbol(enzyme.__nodes__): Array [
    null,
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
